src/applications/static-pages/cta-widget/components/messages/NoMHVAccount.jsx

```diff
@@ -34,6 +48,10 @@ const NoMHVAccount = ({
   return <CallToActionAlert {...content} />;
 };
 
+NoMHVAccount.defaultProps = {
+  recordEventFn: recordEvent,
+};
+
 NoMHVAccount.propTypes = {
```

Comment:
Though `propTypes` is also deprecated, still in use/linted in this project, so I think we want a definition for `recordEventFn` in `propTypes`